pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cognite-neat"
-version = "0.40.0"
+version = "0.40.1"
 readme = "README.md"
```

Comment:
Also set the minimum version of the Python-SDK to  be above `6.39.2` as neat will not function with a lower version.